docs/source/deprecation_page.rst

```diff
@@ -0,0 +1,216 @@
+Deprecation for Kokkos-3.x
+==========================
+
+
+
+Type aliases deprecated in Kokkos-3.7
+-------------------------------------
+``ActiveExecutionMemorySpace``, ``host_execution_space``, ``host_memory_space``, ``host_mirror_space``,``is_array_layout``, ``is_execution_policy``, ``is_execution_space``, ``is_memory_space``, ``is_memory_traits``, ``is_space``, ``Iterate``, ``MDRangePolicy``, ``Rank``, ``SpaceAccessibility``
```

Comment:
```suggestion
``ActiveExecutionMemorySpace``, ``host_execution_space``, ``host_memory_space``, ``host_mirror_space``, ``is_array_layout``, ``is_execution_policy``, ``is_execution_space``, ``is_memory_space``, ``is_memory_traits``, ``is_space``, ``Iterate``, ``MDRangePolicy``, ``Rank``, ``SpaceAccessibility``
```
Otherwise, this doesn't seem to render correctly.